### PR TITLE
Update tg.js v5.4.1 (includes changing tg.js into fil.js)

### PR DESCRIPTION
**Pull request by ghost**
**May i propose changing the name of tg.js and "Tagalog" from the in-game selector into "Filipino". I will be fine with a name like fil.js which is also easier to find and understand.**

### CODE REVIEW

File: fil.js [hunk 1]
```diff
@@ -858,9 +858,9 @@ windows.playerlist.punish Sigurado ka bang nais mong parusahin ang manlalarong s
 item.view.open Buksan ang Model Viewer
 item.view.more Iba pang Impormasyon
 item.sell.cant Hindi mabenta ang item na ito.
-item.sell.cant.krunker Item cant be sold on krunker.
+item.sell.cant.krunker Hindi mabenta ang kagamitan sa Krunker.
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply fil.js:861]: If it's okay to remove it, sure. The original English version specifically mentioned Krunker.

[the remark this answers is not in the document]
Comment (Navsy17) [reply fil.js:861]: So this will be tentative and awaiting other opinions.

File: fil.js [hunk 2]
```diff
@@ -888,39 +888,39 @@ social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
 social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
 social.trades.accept Nais mo bang tanggapin ang Pag-alok na ito?
 stats.airdrops Pakete ng KR
-stats.airdrops.stolen Airdrops Stolen
+stats.airdrops.stolen Ninakaw (airdrops)
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply fil.js:891]: We are not sure if it will fit in the stats page. Leave this for now.

**Review by callmedandann — CHANGES_REQUESTED** [review]
Can we make the "Pag-Logout" and "Pag-logout sa lahat ng sesyon" into "Mag-Logout" and "Mag-logout sa lahat ng sesyon"

File: fil.js [hunk 3]
```diff
@@ -217,10 +217,10 @@ settings.interface.healthColE Nametag Health Color Enemy
 settings.interface.xpBarCol Kulay ng XP Bar
 settings.interface.xpBarCol2 XP Bar Color 2
 settings.interface.xpBarOpac Diin ng XP Bar
-settings.medals.header Medals
-settings.medals.show Show Medals
-settings.medals.classic Classic Medals
-settings.medals.sound Play Medal Sounds
+settings.medals.header Medalya
```
Comment (callmedandann) [inline fil.js:220]: Can we make this: "Mga medalya"

File: fil.js [hunk 4]
```diff
@@ -531,23 +531,23 @@ windows.controls.specFree I-toggle ang Free Cam
...
+windows.controls.taunt1 Panukso (Sipol)
+windows.controls.taunt2 Panukso (Utot)
+windows.controls.taunt3 Panukso (Meme)
+windows.controls.taunt4 Panukso (WTF ano 'to?)
+windows.controls.taunt5 Panukso (Ngayon na!!)
 windows.controls.prop Piliin ang Prop (Hold M1)
-windows.controls.propRand Select Random Prop
-windows.controls.propRot Toggle Rotation
-windows.controls.propRotR Reset Rotation
+windows.controls.propRand Piliin ang kahit anong prop
```
Comment (callmedandann) [inline fil.js:548]: Can we make this: "Pumili ng kahit anong prop"

File: fil.js [hunk 5]
```diff
@@ -670,11 +670,11 @@ mod.error Error sa mod!
 mod.loading Naglo-load ng mod ...
 mod.invalid Imbalidong mod file!
 matchmaker.full PUNO
-matchmaker.GameFull Game is full.
-matchmaker.NoServersMatchQuery GAME UPDATING
-matchmaker.InvalidGameId Game not found.
-matchmaker.NoAvailableServers Servers are at Max Capacity
-matchmaker.GameIssue Unknown Join Issue
+matchmaker.GameFull Puno ang laro.
```
Comment (callmedandann) [inline fil.js:673]: Can we make this: "Puno na ang laro"

File: fil.js [hunk 6]
```diff
@@ -670,11 +670,11 @@ mod.error Error sa mod!
...
+matchmaker.NoServersMatchQuery NAG-UUPDATE ANG LARO
+matchmaker.InvalidGameId Hindi nahanap ang laro.
```
Comment (callmedandann) [inline fil.js:675]: Can we change the "nahanap" into "mahanap"

**Review by callmedandann — CHANGES_REQUESTED** [review]

File: fil.js (hunk 5 above)
Comment (Navsy17) [reply fil.js:673]: It makes it wordy to read. Awaiting opinions from more reviewers.

File: fil.js (hunk 6 above)
Comment (Navsy17) [reply fil.js:675]: Thanks alot.

File: fil.js (hunk 3 above)
Comment (Navsy17) [reply fil.js:220]: Sure.

File: fil.js (hunk 4 above)
Comment (Navsy17) [reply fil.js:548]: Sure, thanks.

**Comment by mty22** [comment]
@Navsy17 for now, keep it as `tg.js`. We can discuss changing it in a different pull request. Can you please update the file name and re-push?

**Comment by mty22** [comment]
@callmedandann also please confirm IGN

**Review by callmedandann — CHANGES_REQUESTED** [review]
153 - Add space between ng & Textura

**Review by callmedandann — CHANGES_REQUESTED** [review]
673 - Can we make it "Puno na ang laro"
675 - Instead of nahanap, can we just make it mahanap?
679 - Make Kolektible into Nakokolekta 
891 - Can we make it "Ninakaw na airdrops"
546 - There are double "!"

**Comment by mty22** [comment]
I've applied a fix, can you please review your respective language and implement? https://github.com/mty22/krunker.io-translations/commit/5ec0f43d446e618fd7d0a446ba2fbcb9138eacd4

**Comment by Navsy17** [comment]
> I've applied a fix, can you please review your respective language and implement? [5ec0f43](https://github.com/mty22/krunker.io-translations/commit/5ec0f43d446e618fd7d0a446ba2fbcb9138eacd4)

**Comment by Navsy17** [comment]
My bad i accidentally closed it

**Comment by mty22** [comment]
@Navsy17 let me know when ready to merge.

**Comment by mty22** [comment]
@Navsy17 did you close this by mistake?

**Comment by mty22** [comment]
Actually, I'll probably need to push to a new branch since this one has issues.

**Pull request closed without merging**